This patch comes from upstream; it is the essential change between 2.5.15
and 2.5.16; once we upgrade to 2.5.16 or later, the patch can be removed.
The patch is needed to get kombu to install properly with Python 3.

--- kombu-2.5.12/setup.py	2013-06-28 07:11:26.000000000 -0700
+++ kombu-2.5.12/setup.py	2013-10-23 10:53:25.000000000 -0700
@@ -108,8 +108,8 @@
 
 
 def reqs(f):
-    return filter(None, [strip_comments(l) for l in open(
-        os.path.join(os.getcwd(), 'requirements', f)).readlines()])
+    return list(filter(None, [strip_comments(l) for l in open(
+        os.path.join(os.getcwd(), 'requirements', f)).readlines()]))
 
 install_requires = reqs('default.txt')
 if py_version[0:2] == (2, 6):
